include/irr/core/math/aabb2dSIMD.h

```diff
@@ -8,90 +8,92 @@
 #include <iostream>
 
 #define SIGN_FLIP_MASK_XY	core::vector3du32_SIMD(0x80000000, 0x80000000, 0x00000000, 0x00000000)
-#define SIGN_FLIP_MASK_XYZW core::vector3du32_SIMD(0x80000000, 0x80000000, 0x80000000, 0x80000000)
-
 
 namespace irr { namespace core {
 
+
 class aabb2dSIMDf
```

Comment:
this needs to inherit from a AlignedBase class to ensure alignment and `operator new`+`operator delete` overrides

Reply:
same for the other vector type